src/start/DevAuthScreen.js

```diff
@@ -111,7 +111,11 @@ class DevAuthScreen extends PureComponent<Props, State> {
             keyExtractor={item => `${item.email}${item.realm_uri}`}
             ItemSeparatorComponent={() => <View style={componentStyles.accountItem} />}
             renderItem={({ item }) => (
-              <ZulipButton text={item.email} secondary onPress={() => this.tryDevLogin(item)} />
+              <ZulipButton
```

Comment:
That looks weird visually.
What is the reason to want to add the realm url?

Reply:
Because same user with same email can have account on multiple realm.
like `Cordelia@zulip.com`

Reply:
How about the realm's short name instead?

Perhaps in parens, and left out if it's the main realm. So like "cordelia@zulip.com (lear)".

Reply:
Updated 👍